code/datums/emotes.dm

```diff
@@ -265,6 +265,18 @@
  * Returns a bool about whether or not the user should play a sound when performing the emote.
  */
 /datum/emote/proc/should_play_sound(mob/user, intentional = FALSE)
+	if(emote_type == EMOTE_AUDIBLE)
+		if(!muzzle_ignore && user.is_muzzled())
+			return FALSE
+		if(HAS_TRAIT(user, TRAIT_MUTE))
+			return FALSE
```

Comment:
What I'm getting at is people with TRAIT_MUTE won't be able clap, as the [clap emote](https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/carbon/emote.dm#L18) is EMOTE_AUDIBLE, right? 

I might be misunderstanding something though

Reply:
Fixed with c0487d2aa46207aab8c4030d0a636354c139e4bc